### PR TITLE
Feature/hn 4 add persistence to the project

### PR DESCRIPTION
**Pull request by alexsandermeneses**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,6 +48,33 @@
...
+
+		<dependency>
+			<groupId>org.postgresql</groupId>
+			<artifactId>postgresql</artifactId>
+			<scope>runtime</scope>
+			<version>42.6.0</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.data</groupId>
```
Comment (arnaldotecadm) [inline pom.xml:65]: This dependency is duplicated

File: target/classes/postman/CollectionPostman.postman_collection.json
Comment (arnaldotecadm) [inline target/classes/postman/CollectionPostman.postman_collection.json:1]: Please add the folder target to gitignore.

File: src/main/java/com/happeningnow/Application.java [hunk 2]
```diff
@@ -8,6 +8,7 @@
 public class Application {
 
 	public static void main(String[] args) {
+		//System.out.println("Hello Word");
```
Comment (arnaldotecadm) [inline src/main/java/com/happeningnow/Application.java:11]: Remove unused comments

File: src/main/resources/application.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+spring:
+  datasource:
+    url: jdbc:postgresql://localhost:5432/happening-now
```
Comment (arnaldotecadm) [inline src/main/resources/application.yaml:3]: Use environment variables in order to receive these values.
It goes for: Hostname, Port, Username and Password

**Review by arnaldotecadm — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (alexsandermeneses) [reply pom.xml:65]: ok

File: target/classes/postman/CollectionPostman.postman_collection.json
Comment (alexsandermeneses) [reply target/classes/postman/CollectionPostman.postman_collection.json:1]: ok

**Review by arnaldotecadm — APPROVED** [review]
LGTM

**Pull request merged**